conf/test.config

```diff
@@ -15,14 +15,22 @@ params {
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
     // Limit resources so that this can run on GitHub Actions
-    max_cpus   = 2
-    max_memory = '6.GB'
-    max_time   = '6.h'
+    max_cpus                   = 2
+    max_memory                 = '6.GB'
+    max_time                   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'assets/samplesheet_v1.csv'
+    //input  = '../final_test_data/samplesheet_v3.csv'
+    input                         = 'assets/samplesheet_v1.csv'
+
+    taxid                         = '211044 11676'         //separated by space
```

Comment:
Shall a file with taxids be included in the tests as well?

Reply:
I thought that a user would be able to supply a file with taxids. Or am I wrong?